### DISCUSSION TITLE
display surprise on admin dashboard

### DISCUSSION THREAD

**Discussion by hit7374** [joomla/joomla-cms#32303] [Q&A] [answered]
### Steps to reproduce the issue
menu items are framed with ** or ?? after updating from j4beta 6 to beta7

**Comment by brianteeman** [comment, marked as answer]
You have language debug enabled

**Comment by hit7374** [comment]
Thanks
it works now<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32302">issues.joomla.org/tracker/joomla-cms/32302</a>.</sub>
